### ISSUE TITLE
[Question] FF6Tools to Ips Patch

### ISSUE THREAD

**Issue by shinra358** [everything8215/ff6tools#22]
is there a way to convert a patch made in ff6tools to an ips patch?

**Comment by everything8215** [comment]
FF6Tools doesn’t create patches, it saves ROM files. To make a patch, you can use any patch generation utility with the original ROM file as the base file and the modified ROM file as the patched file.

**Comment by shinra358** [comment]
.ymls can be saved so that someone else could apply to their rom. that's technically a patch and i was wondering if .ymls can be converted into ips's.

**Comment by everything8215** [comment]
No, not directly. The yml files can only be used within FF6Tools.

**Comment by shinra358** [comment]
oh ok. im just worried about the future of this since it's online only.
